client/src/ShareButton.css

```diff
@@ -0,0 +1,3 @@
+.chalk {
+    background-image: url(https://i.ibb.co/Vjhn6P7/chalkboard-background1.jpg);
```

Comment:
As this is an external link I suspect this will not work on heroku and will have the same content-security-policy issue as the other images. It's not a worry for now though as it should be fixed as a result of the other story to address the policy

Reply:
My bad :)